### PR TITLE
cell_properties tests added

### PR DESCRIPTION
**Pull request by maxm17**

### CODE REVIEW

File: tests/unit_tests/grid/test_cell_properties.py [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+    if hasattr(grid, 'array_thickness'):
+        delattr(grid, 'array_thickness')
+    if hasattr(grid, 'property_collection'):
+        delattr(grid, 'property_collection')
+
+    # Act
+    thickness = cp.thickness(grid)
+
+    # Assert
+    np.testing.assert_array_almost_equal(thickness, 20.0)
```
Comment (cflynn3) [inline tests/unit_tests/grid/test_cell_properties.py:125]: Surprised assert_array_almost_equal works comparing against a float?

File: tests/unit_tests/grid/test_cell_properties.py [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+    if hasattr(grid, 'array_volume'):
+        delattr(grid, 'array_thickness')
+    if hasattr(grid, 'property_volume'):
+        delattr(grid, 'property_collection')
+
+    # Act
+    volume = cp.volume(grid)
+
+    # Assert
+    np.testing.assert_array_almost_equal(volume, 100000.0)
```
Comment (cflynn3) [inline tests/unit_tests/grid/test_cell_properties.py:231]: As above, I think a different assert method should be used.

**Review by cflynn3 — CHANGES_REQUESTED** [review]
Looks good, but please fix the two highlighted assert methods.

File: tests/unit_tests/grid/test_cell_properties.py (hunk 1 above)
Comment (andy-beer) [reply tests/unit_tests/grid/test_cell_properties.py:125]: numpy has the concept of 'array like' which will take a scalar and treat it as a constant array when needed, if I've understood correctly, so I prefer the form Max has used here

Comment (cflynn3) [reply tests/unit_tests/grid/test_cell_properties.py:125]: OK, in that case I retract my comments :-).

**Review by cflynn3 — APPROVED** [review]

**Pull request merged**